package.json

```
@@ -14,6 +14,7 @@
   "scripts": {
     "fasttest": "IMAGE_NAME=test-open-balena-api-fast ./automation/fasttest.sh",
     "fasttest-cleanup": "IMAGE_NAME=test-open-balena-api-fast ./automation/fasttest.sh --teardown",
+    "generate-model-types": "npm run fasttest -- --generate-config .materialized-config.json && echo '/**\n * This file is auto-generated with `npm run generate-model-types`\n */\n' > src/balena-model.ts && npx abstract-sql-compiler generate-types .materialized-config.json >> src/balena-model.ts",
```

I'd maybe make this
```suggestion
    "generate-model-types": "npm runmaterialize-config && echo '/**\n * This file is auto-generated with `npm run generate-model-types`\n */\n' > src/balena-model.ts && npx abstract-sql-compiler generate-types .materialized-config.json >> src/balena-model.ts",
    "materialize-config": "npm run fasttest -- --generate-config .materialized-config.json",
```
since there are benefits to materializing the config by itself, eg for `npx odata-compiler`